static/js/pivotal.js

```diff
@@ -143,21 +173,78 @@
         'X-TrackerToken': pt_token
       },
       success: function(data) {
-        // Find all git commit messages
-        var list = data.filter(function(obj) {
-          return obj.commit_type;
-        });
-        // Find repo name from comment body text
-        list = list.map(function(item) {
-          return item.text.split('https://github.com/gengo/')[1].split('/commit/')[0];
+        var PULL_REQUEST_REGEX = /Merge pull request/;
+        var COMMIT_REPO_REGEX = /https:\/\/github.com\/gengo\/(.+)\/commit\//;
+        var DEPLOY_REPO_REGEX = new RegExp('Deployed (.+) to '+ config.environment +': ');
+
+        var activities = data.filter(function(activity) {
+          return activity.commit_type === 'github' || DEPLOY_REPO_REGEX.test(activity.text);
+        }).reverse();
+
+        var activitiesGrouped = groupBy(activities, function(activity) {
```

Comment:
I think it's valuable to add by what it's grouped so something like `activitiesGroupedByRepo`, or shorter: `activitiesByRepo`. What do you think ?